releases/xrpl/2025-02-v1.0.0.md

```diff
@@ -0,0 +1,205 @@
+# XRPL v1.0.0
+
+|                | **Owner**                        |
+| -------------- | -------------------------------- |
+| **Created By** | @k4m4 <nikolas@commonprefix.com> |
+| **Deployment** | @k4m4 <nikolas@commonprefix.com> |
+
+| **Network**          | **Deployment Status** | **Date**   |
+| -------------------- | --------------------- | ---------- |
+| **Devnet Amplifier** | Deployed              | 2025-01-29 |
+| **Stagenet**         | -                     | TBD        |
+| **Testnet**          | -                     | TBD        |
+| **Mainnet**          | -                     | TBD        |
+
+## Background
+
+Changes in the release:
+
+This is the v1.0.0 initial release.
+
+## Deployment
+
+Create an `.env` config. `CHAIN` should be set to `xrpl` for mainnet, and `xrpl-2025-q1` for all other networks.
+
+```yaml
+SEED=xyz
+ENV=xyz
+CHAIN=xyz
+```
+
+An initial XRPL chain config needs to be added to `${ENV}.json` file under `chains` key.
+
+#### Devnet-Amplifier
+
+```bash
+\"$CHAIN\": {
+  "name": "XRPL",
+  "axelarId": \"$CHAIN\",
+  "rpc": "wss://s.devnet.rippletest.net:51233",
+  "tokenSymbol": "XRP",
+  "networkType": "testnet",
+  "chainType": "xrpl",
+  "finality": "1",
+  "approxFinalityWaitTime": 1,
+  "explorer": {
+    "name": "XRPL Explorer",
+    "url": "https://devnet.xrpl.org"
+  }
+}
+```
+
+#### Stagenet / Testnet
+
+```bash
+\"$CHAIN\": {
+  "name": "XRPL",
+  "axelarId": \"$CHAIN\",
+  "rpc": "wss://s.altnet.rippletest.net:51233",
+  "tokenSymbol": "XRP",
+  "networkType": "testnet",
+  "chainType": "xrpl",
+  "finality": "1",
+  "approxFinalityWaitTime": 1,
+  "explorer": {
+    "name": "XRPL Explorer",
+    "url": "https://testnet.xrpl.org"
+  }
+}
+```
+
+#### Mainnet
+
+```bash
+"xrpl": {
+  "name": "XRPL",
+  "axelarId": "xrpl",
+  "rpc": "wss://s1.ripple.com:51233",
+  "tokenSymbol": "XRP",
+  "networkType": "mainnet",
+  "chainType": "xrpl",
+  "finality": "1",
+  "approxFinalityWaitTime": 1,
+  "explorer": {
+    "name": "XRPL Explorer",
+    "url": "https://livenet.xrpl.org"
+  }
+}
+```
+
+1. Request tokens from faucet
+
+```bash
+node xrpl/faucet.js
+```
+
+2. Deploy XRPL Multisig account (edge AxelarGateway equivalent)
+
+```bash
+node xrpl/deploy-multisig.js --initialSigners r4LDyA6WoyzJ6ZhQ5NWU4AG1aApBwbSSr4LDyA6WoyzJ6ZhQ5NWU4AG1aApBwbSSk6
+```
+
+3. After creating the XRPL Multisig, ensure that you deploy the [XRPL Amplifier contracts](../cosmwasm/2025-02-XRPL-v1.0.0.md)
+
+4. Set genesis verifier set
+
+```bash
+axelard tx wasm execute $XRPL_MULTISIG_PROVER '"update_verifier_set"' "${ARGS[@]}"
```

Comment:
Does the XRPL gateway signer list need to be updated with the genesis verifier set after this step? Don't see where that's being done. We don't have the full verifier set at the time of the multisig address creation (since verifiers haven't registered their keys yet until XRPL amplifier contracts are deployed)

Reply:
On second thought, we can just set some 'deployer' address as the initial signer and make sure to set the XRPLMultisigProver's `nextSequenceNumber` to `actualNextSequenceNumber + 1`, so that the deployer can perform the handover to the actual validator set when the time comes.